src/ScrumMaster/Command/Exception/UndefinedParameter.php

```diff
@@ -6,8 +6,8 @@
 
 final class UndefinedParameter extends \Exception
 {
-    public function __construct(string $parameterName)
+    public function __construct(array $parameterName)
```

Comment:
This is not a single value anymore. Please, change it to `$parameterNames` or `$parameters` 😃